Assets/MRTK/Services/CameraSystem/CameraSystem.asmdef.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
+guid: a921677c6fdd77c468449f41a792ec1b
```

Comment:
This used to be Microsoft.MixedReality.Toolkit.Services.CameraSystem.asmdef.meta right?

It looks like the guid changed here

Reply:
nice catch.... i'll fix the guid